src/WebJobs.Extensions.DurableTask/Options/DurableTaskOptions.cs

```diff
@@ -203,6 +203,11 @@ public string HubName
         /// </summary>
         public bool UseAppLease { get; set; } = true;
 
+        /// <summary>
+        /// If IncludeParameters is true, the input of the activity function will be displayed.
+        /// </summary>
+        public bool IncludeParameters { get; set; } = false;
```

Comment:
Let's rename this to `StoreInputsInOrchestrationHistory`. This is a better description of what the feature does compared to "IncludeParameters", which is vague.